jupyter-js-widgets/css/widgets-base.css

```diff
@@ -376,13 +379,15 @@
 .widget-text input[type="text"], .widget-textarea textarea {
     box-sizing: border-box;
     border: var(--jp-widgets-input-border-width) solid var(--jp-widgets-input-border-color);
-    background-color: var(--jp-widgets-input-backgroundColor);
+    background-color: var(--jp-widgets-input-background-color);
     color: var(--jp-widgets-input-color);
     font-size: var(--jp-widgets-font-size);
-    padding: var(--jp-widgets-input-padding);
+    padding: var(--jp-widgets-input-padding) calc( var(--jp-widgets-input-padding) *  2 );
```

Comment:
@ellisonbg 

This change does not seem to be correctly parsed, resulting in `padding` not being set.

A consequence is that dropdowns have a padding issue in readthedocs

thathttp://ipywidgets.readthedocs.io/en/latest/examples/Widget%20List.html